requirements.txt

```diff
@@ -8,4 +8,5 @@ python-glanceclient
 python-cinderclient
 python-neutronclient
 python-keystoneclient
+python-ironicclient
```

Comment:
I believe it will be used on the local system in the same way that the other python clients are -- to communicate to the ironic-api service, once it is set up, for things like inventory, machine management, etc. Note that this is needed for OSC to provide the "baremetal" set of commands, for shade to talk to ironic, etc.

If there's somewhere else this should get added...